flake.nix

```diff
@@ -22,14 +22,15 @@
         "x86_64-linux"
       ];
       imports = [
+        inputs.devenv.flakeModule
         inputs.flake-root.flakeModule
```

Comment:
Is flake-root redundant now that we use devenv? IIRC it is something used by the mission-control.

Reply:
This seems to be the case, there is [no mention](https://flake.parts/options/mission-control) of mission-control needing the flake-root, at least not anymore. To be honest, I don't even know what needed this, but I removed it and everything seems to be fine.. so yeah.